sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultKeyStore.java

```diff
@@ -271,7 +287,8 @@ public void engineSetKeyEntry(String alias, byte[] key, Certificate[] chain) {
 
     @Override
     public int engineSize() {
-        return aliases != null ? aliases.size() : 0;
+        List<String> portalAliases = keyVaultCertificates.getAliases();
+        return (portalAliases != null ? portalAliases.size() : 0) + classpathCertificates.getAliases().size();
```

Comment:
It's weird that `keyVaultCertificates.getAliases()` need null check but classpathCertificates does not need null check.